dotCMS/src/main/java/com/dotmarketing/portlets/osgi/AJAX/OSGIAJAX.java

```diff
@@ -32,7 +32,7 @@ public void action ( HttpServletRequest request, HttpServletResponse response )
     public void undeploy ( HttpServletRequest request, HttpServletResponse response ) throws ServletException, IOException, InterruptedException {
         validateUser() ;
         
-        String jar = request.getParameter( "jar" );
+        String jarName = request.getParameter( "jar" );
```

Comment:
Call `FileUtil.sanitizeFileName` on this value before using it.

Reply:
done. Good call.